ext/fetch/lib.rs

```diff
@@ -316,12 +321,16 @@ impl FetchPermissions for deno_permissions::PermissionsContainer {
   }
 
   #[inline(always)]
-  fn check_read(
+  fn check_read<'a>(
```

Comment:
Add `must_use` attr as well?

Reply:
It's on the trait. I think it doesn't need to be here as well?